### ISSUE TITLE
Entity Framework Core support

### ISSUE THREAD

**Issue by biggery** [blackcity/Backload#112]
We're using Backload in our main product with local file system, SQL Server, Azure Blob and Azure SQL Server as storage targets. So, we are very pleased with it. 
Currently we are planning to move to ASP.NET Core resp. Entity Framework Core. We developed a very basic demo application to figure out what problems we may get. 
Backload supports .NET Core, but currently there is no EF Core support available. Do you plan to publish database support and if so, what is your timeframe?

**Comment by Auersberg** [comment]
We already added EF Core support but it's not published yet. While I would say there are no problems compared to the .NET version since they share 99.9% of the code base,  we need to add more module tests for the 2.2.8 release.

**Comment by biggery** [comment]
That's great! Is database access also working on a Linux server or Mac?

**Comment by Auersberg** [comment]
Yes, it runs on Linux and Mac. We will provide a developer package for all platforms. You can access all supported databases like SQL Server, incl. Azure SQL Server, SqlLite, Oracle etc. LocalDB is not supported on Linux or Mac since it is currently not available on non Windows platforms.
